src/win/error.c

```diff
@@ -102,6 +102,7 @@ int uv_translate_sys_error(int sys_errno) {
     case ERROR_INSUFFICIENT_BUFFER:         return UV_EINVAL;
     case ERROR_INVALID_DATA:                return UV_EINVAL;
     case ERROR_INVALID_PARAMETER:           return UV_EINVAL;
+    case ERROR_NOT_A_REPARSE_POINT:         return UV_EINVAL;
```

Comment:
I'm not 100% sure a blanket mapping is okay. I'd maybe play it safe and handle ERROR_NOT_A_REPARSE_POINT explicitly in uv_fs_readlink().

Reply:
Google gives me the impression that this error also might happen when trying to create a reparse point someone invalid (like a network share or onedrive or on FAT). On linux, that cause of this error should map to EPERM:

       EPERM  The filesystem containing oldpath and newpath does not support the creation of hard links.

Reply:
@bnoordhuis @vtjnash good point, done. PTAL!